templates/gitlab-ci-template-cloud-run.yml

```diff
@@ -164,9 +146,16 @@ test variables:
 build:
   stage: build
   extends: .build_template
+  artifacts:
+    expire_in: 1 day
```

Comment:
@Syphon83 just wondering if we can keep the value lower than this ? how the garbage collection is managed on the gitlab side ?

Reply:
Artifacts smallest granularity is seconds. 
`artifacts:
    expire_in: 30 seconds`

If no expiring date has been declared there's an instance wide value in the admin area. 

By default the most recent artifact is always kept if the pipeline succeded.

Reply:
we can set a lower value, i think that `1 hour` is more than enough